mantle/platform/qemu.go

```diff
@@ -1111,6 +1111,11 @@ func baseQemuArgs() []string {
 	}
 	if kvm {
 		ret = append(ret, "-cpu", "host")
+	} else {
+		// the default qemu64 CPU model does not support x86_64_v2
+		// causing crashes on EL9+ kernels
+		// see https://bugzilla.redhat.com/show_bug.cgi?id=2060839
+		ret = append(ret, "-cpu", "Nehalem")
```

Comment:
This will break on non-x86_64 architectures.  I guess for now it's probably simplest to do `if system.RpmArch() == "x86_64"` here.